phpBB/styles/prosilver/template/posting_editor.html

```diff
@@ -183,6 +183,9 @@
 			</dl>
 			<!-- ENDIF -->
 		</fieldset>
-		<!-- ENDIF -->
+
+		</div>
```

Comment:
Well with your patch I get
```
Line 417, Column 7: End tag div seen, but there were open elements.
	</div>
```
on http://validator.w3.org/#validate_by_input
So it seems like you are wrong ;)